### ISSUE TITLE
`simplify('-1 / (-x)')` expected `1/x` but `-(1 / -x)`

### ISSUE THREAD

**Issue by QuadFlask** [josdejong/mathjs#2156]
```js
math.simplify('1/-x').toTex()
// "\frac{1}{- x}" // correct

math.simplify('-1/x').toTex()
// "-\frac{1}{ x}" // correct

math.simplify('(-1)/(-2)').toTex()
// "\frac{1}{2}" // correct

math.simplify('-1 / (-x)').toTex()
// "-\frac{1}{- x}" // wrong. expected \frac{1}{x}

math.simplify('(-1)/(-x)').toTex()
// "-\frac{1}{- x}" // wrong. expected \frac{1}{x} 
```

**Comment by josdejong** [comment]
Would be nice to improve on this, thanks for your input @QuadFlask 

This issue _may_ be related to #1406. There are a couple more open issues to improve the rules of `simplify`.

Anyone interested in looking into this?

**Comment by ony3000** [comment]
I'm trying to fix this.
However, I have a question in the example above.
Is there any reason why the simplification results of `1/-x` and `-1/x` should be different?

**Comment by josdejong** [comment]
Thanks @ony3000 for picking this up!

I would say it would be nicest if the outcome is standardized one way, like always put the minus on the numerator.

**Comment by josdejong** [comment]
The fix made by @ony3000 is published now in `v10.0.2`. Thanks again.
